packages/api/src/hooks/useGetPasskeysList.ts

```diff
@@ -0,0 +1,19 @@
+import useQuery from '../useQuery';
+import useAuthorize from './useAuthorize';
```

Comment:
@yauheni-deriv I dont think we can use useAuthorize in accounts package yet. It will log you out since accounts package still uses mobx

Reply:
will move it to hooks as per our discussion with Yashim